packages/core/src/tracing/instrumentation/cloud/azure/blob.js

```diff
@@ -0,0 +1,153 @@
+/*
+ * (c) Copyright IBM Corp. 2023
+ */
+
+'use strict';
+
+const shimmer = require('../../../shimmer');
+
+const requireHook = require('../../../../util/requireHook');
+const tracingUtil = require('../../../tracingUtil');
+const constants = require('../../../constants');
+const cls = require('../../../cls');
+
+let isActive = false;
+
+exports.spanName = 'azstorage';
+
+exports.init = function init() {
+  requireHook.onModuleLoad('@azure/storage-blob', instrumentBlob);
+};
+
+function instrumentBlob(blob) {
+  instrumentBlockBlob(blob.BlockBlobClient);
+  instrumentBlobClient(blob.BlobClient);
+}
+
+// For the download and delete functionality applicable to block blobs, the BlockBlobClient extends BlobClient
```

Comment:
Qs: Does the instrumentation only handle the following operations: upload, delete, download, and stageBlock?

Reply:
yes. As of now it covers only the upload, delete and download ops. stageblock is one another method to upload